pkgs/development/haskell-modules/configuration-ghc-8.4.x.nix

```diff
@@ -64,12 +65,45 @@ self: super: {
   stack = (doJailbreak super.stack).override {
     Cabal = self.Cabal_2_4_1_0;
     hpack = self.hpack_0_31_1.override { Cabal = self.Cabal_2_4_1_0; };
-    yaml = self.yaml_0_11_0_0;
     hackage-security = self.hackage-security.override { Cabal = self.Cabal_2_4_1_0; };
   };
-  hpack_0_31_1 = super.hpack_0_31_1.override {
-    yaml = self.yaml_0_11_0_0;
-  };
+  hpack_0_31_1 = self.callPackage
```

Comment:
Same here. Why did you add that build expression into the overrides file?

Reply:
Removed.